### DISCUSSION TITLE
Add a short issue body when generating issues from task lists

### DISCUSSION THREAD

**Discussion by choldgraf** [community/community#4380] [Projects and Issues]
The ability to generate issues from task lists is an excellent feature ✨ 

One way in which I think it could be used more seamlessly is to make it possible to generate a "top comment" in the issue when it is created. For example, with a small window that lets you type this in from the task list.

This wouldn't be suitable for really long/complex issue content, but could be useful for providing a quick sentence or two to contextualize the new issue, and avoid the awkward "empty top comment" effect. Perhaps there could be a maximum character limit of the pop-up window of, say, 500 characters.

**Comment by abernix** [comment]
What if, rather than a popup dialog, the way you did this was just using multiple lines within the existing Markdown checkbox syntax?:

```markdown
- [ ] Title of item

      And indenting the content you want to become the body, much in
	  the same format as a Git commit where the first line is the message/title
      and subsequent lines after a blank line are the body of the issue?
- [ ] Title of next item

      And another body here.
- [ ] And an item without a body.
```

Overall, I wouldn't be against a popup dialog (perhaps to confirm the title is as you hope), but this would seem to make it possible to riff/iterate on content in a parent issue before breaking it down further.
